Configurateur/front/lib/screens/user-list/user_list.dart

```diff
@@ -105,7 +105,7 @@ class _UserPageState extends State<UserPage> {
 
   Future<void> fetchUserMobile() async {
     final response = await http
-        .get(Uri.parse('http://${serverIp}:3000/api/mobile/user/listAll'));
+        .get(Uri.parse('http://${serverIp}:8080/api/dev/user/listall'));
```

Comment:
server is on port 3000 so why switch to 8080 ?

Reply:
I put it back